scripts/material_map/material_mapping_epic.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2024 Shujie Li
+
+import os
+import argparse
+
+import acts
+from acts.examples import JsonFormat
+
+import epic
+from material_mapping import runMaterialMapping
+
+if "__main__" == __name__:
+
+    p = argparse.ArgumentParser(
+        description="Script to generate material map for ePIC geometry"
+    )
+    p.add_argument(
+        "--xmlFile",
+        default=os.environ.get("DETECTOR_PATH", "")+"epic_craterlake.xml",
+        help="input xml file containing ePIC geometry",
+    )
+    p.add_argument(
+        "--geoFile",
+        type=str,
+        default="geometry-map.json",
+        help="input json file to define volumes and layers used in material mapping",
+    )
+    p.add_argument(
+        "--matFile",
+        type=str,
+        default="material-map.json",
+        help="output filename for the generated material map, can be json and cbor formats",
```

Comment:
With the default ACTS material mapping function, the output is set to be json format which is x10 larger than cbor. But otherwise it's ok...

Reply:
Understood. Thank you for upstream https://github.com/acts-project/acts/pull/3071.